cpp/models/abm/person.h

```diff
@@ -481,15 +481,15 @@ class Person
     Mask m_mask; ///< The Mask of the Person.
     std::vector<ScalarType>
         m_compliance; ///< Vector of compliance values for all #InterventionType%s. Values from 0 to 1.
-    PersonId m_person_id; ///< Id of the Person. Corresponds to the index in m_persons in Model.
     std::vector<uint32_t> m_cells; ///< Vector with all Cell%s the Person visits at its current Location.
     mio::abm::TransportMode m_last_transport_mode; ///< TransportMode the Person used to get to its current Location.
     Counter<uint32_t> m_rng_counter{0}; ///< counter for RandomNumberGenerator.
     CustomIndexArray<TestResult, TestType> m_test_results; ///< CustomIndexArray for TestResults.
     std::vector<int>
         m_assigned_location_model_ids; ///< Vector with model ids of the assigned locations. Only used in graph abm.
-    uint64_t
-        m_unique_id; ///< Unique identifier of a person. Is only relevant in graph abm, otherwise is equal to m_person_id.
+    PersonId m_person_id; ///< Unique identifier of a person.
+    uint32_t m_rng_index; ///< Index for PersonalRandomNumberGenerator.
+    mio::Key<uint64_t> m_rng_key; ///< Key for PersonalRandomNumberGenerator
```

Comment:
I think it would be good to group these two with the rng_counter in the order key, index, counter, to have better semantic ordering.